test/cases/postSkipSegments.js

```diff
@@ -88,18 +88,48 @@ describe('postSkipSegments', () => {
           done("Status code was " + res.statusCode);
         }
       });
-  });
+  }).timeout(5000);
 
   it('Should be rejected if over 80% of the video', (done) => {
-    request.get(utils.getbaseURL() 
-     + "/api/postVideoSponsorTimes?videoID=qqwerty&startTime=30&endTime=1000000&userID=testing", null, 
+    request.get(utils.getbaseURL()
+     + "/api/postVideoSponsorTimes?videoID=qqwerty&startTime=30&endTime=1000000&userID=testing", null,
       (err, res, body) => {
         if (err) done("Couldn't call endpoint");
         else if (res.statusCode === 403) done(); // pass
         else done("non 403 status code: " + res.statusCode + " ("+body+")");
       });
   });
 
+  it("Should be rejected if there's not at least 65% overlap with NB", (done) => {
+    request.get(utils.getbaseURL()
+     + "/api/postVideoSponsorTimes?videoID=LevkAjUE6d4&startTime=40&endTime=60&userID=testing", null,
+      (err, res, body) => {
+        if (err) done("Couldn't call endpoint");
+        else if (res.statusCode === 403) done(); // pass
+        else done("non 403 status code: " + res.statusCode + " ("+body+")");
+      });
+  });
+
+  it("Should be accepted if only off by 7s", (done) => {
+    request.get(utils.getbaseURL()
+     + "/api/postVideoSponsorTimes?videoID=LevkAjUE6d4&startTime=0&endTime=14.079&userID=testing", null,
+      (err, res, body) => {
+        if (err) done("Couldn't call endpoint");
+        else if (res.statusCode === 200) done(); // pass
+        else done("non 403 status code: " + res.statusCode + " ("+body+")");
+      });
+  });
+
+  it("Should be accepted if there's at least 65% overlap with NB" , (done) => {
+    request.get(utils.getbaseURL()
+     + "/api/postVideoSponsorTimes?videoID=LevkAjUE6d4&startTime=0&endTime=6&userID=testing", null,
+      (err, res, body) => {
+        if (err) done("Couldn't call endpoint");
+        else if (res.statusCode === 200) done(); // pass
+        else done("non 200 status code: " + res.statusCode + " ("+body+")");
```

Comment:
should be res.statusCode rather than body if the reason is failed is a status code check